bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_rooglobalfunc.py

```diff
@@ -0,0 +1,159 @@
+# Authors:
+# * Jonas Rembser 06/2021
+# * Harshal Shende 06/2021
+
+################################################################################
+# Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+
+r"""
+/**
+\class RooGlobalFunc
+\brief \parblock \endparblock
+\htmlonly
+<div class="pyrootbox">
+\endhtmlonly
+
+## PyROOT
+
+Some member functions of RooGlobalFunc that take a RooCmdArg as argument also support keyword arguments.
+So far, this applies to FitOptions, Format, Frame, MultiArg, YVar and ZVar.
+\code{.py}
+# Directly passing a RooCmdArg:
+ROOT.RooMCStudy(model, ROOT.RooArgSet(x), ROOT.RooFit.FitOptions=(ROOT.RooFit.Save(True), ROOT.RooFit.PrintEvalErrors(0)))
+
+# With keyword arguments:
+ROOT.RooMCStudy(model, ROOT.RooArgSet(x), FitOptions=dict(Save=True, PrintEvalErrors=0))
+\endcode
+
+\htmlonly
+</div>
+\endhtmlonly
+*/
+"""
+
+from ._utils import _kwargs_to_roocmdargs, _string_to_root_attribute
+
+
+# Color and Style dictionary to define matplotlib conventions
+_color_map = {
+    "r": "kRed",
+    "b": "kBlue",
+    "g": "kGreen",
+    "y": "kYellow",
+    "w": "kWhite",
+    "k": "kBlack",
+    "m": "kMagenta",
+    "c": "kCyan",
+}
+_style_map = {"-": "kSolid", "--": "kDashed", ":": "kDotted", "-.": "kDashDotted"}
+
+
+def FitOptions(*args, **kwargs):
+    # Redefinition of `FitOptions` for keyword arguments.
+    # The keywords must correspond to the CmdArg of the `FitOptions` function.
+    from cppyy.gbl import RooFit
+
+    args, kwargs = _kwargs_to_roocmdargs(*args, **kwargs)
+    return RooFit._FitOptions(*args, **kwargs)
+
+
+def Format(*args, **kwargs):
+    # Redefinition of `Format` for keyword arguments.
+    # The keywords must correspond to the CmdArg of the `Format` function.
+    from cppyy.gbl import RooFit
+
+    if "what" in kwargs:
+        args = (kwargs["what"],) + args
+    del kwargs["what"]
```

Comment:
wong indentation (before it was correct, should be one level more) also in `YVar` and `ZVar`.